src/libs/antares/series/include/antares/series/series.h

```diff
@@ -86,6 +86,7 @@ class TimeSeries
     bool forceReload(bool reload = false) const;
     void markAsModified() const;
     uint64_t memoryUsage() const;
+    void resizeTSNoDataLoss(uint width);
```

Comment:
I don't see the point of turning the old free function **resizeMatrixNoDataLoss** into a new data member function **resizeTSNoDataLoss**. 
To me, it's not necessary : we apparently need it **only** for hydro TS (so it's a implementation detail particular to hydro), and it makes generic class **TimeSeries** interface more complex.